site/layouts/blog/single.html

```diff
@@ -66,6 +66,12 @@ <h6>Other Posts</h6>
                         {{ end }}
                         {{ end }}
                     </ul>
+                    <h6>Categories</h6>
+                    <ul class="list-unstyled">
+                        {{ range .Site.Taxonomies.categories }}
```

Comment:
It seems like every blog post now has lists every category. Could you please take a look?
![Screenshot from 2020-10-22 11-36-51](https://user-images.githubusercontent.com/16541325/96853698-f3831b80-145a-11eb-9f63-d8744129e64f.png)

Reply:
I thought it to would be good to show all of them. I'll revert.

Reply:
And yes, indexes/category.html can be removed.